### ISSUE TITLE
[[inputs.disk]], gopsutil v2.19.7 breaks plugin when run in docker

### ISSUE THREAD

**Issue by i-prudnikov** [influxdata/telegraf#6898]
### Relevant telegraf.conf:
```toml
[[inputs.disk]]
 ignore_fs = ["tmpfs", "devtmpfs", "devfs", "overlay", "aufs", "squashfs"]
```

### System info:
Telegraf version 1.13.0

How to reproduce:
Run telegraf in  docker container (debian 9) with all necessary mounts and env. vars (HOST_MOUNT_PREFIX, HOT_SYS, HOST_PROC,HOST_ETC)

### Expected behavior:
Plugin reports disk statistics

### Actual behavior:
Plugin reports error: `lstat can't stat /dev/mapper`

### Additional info:
The problem comes from https://github.com/shirou/gopsutil/blob/v2.19.7/disk/disk_linux.go#L303.
In v2.19.7, the following code added, that breaks the functionality of the plugin when it run inside container and trying to track host disks:
```go
if strings.HasPrefix(d.Device, "/dev/mapper/") {
	devpath, err := filepath.EvalSymlinks(d.Device)
	if err != nil {
		return nil, err
	}
	d.Device = devpath
}
```
The problem is with this line (https://github.com/shirou/gopsutil/blob/v2.19.7/disk/disk_linux.go#L304):
`devpath, err := filepath.EvalSymlinks(d.Device)`, as it evals path to device not respecting the case if filesystem is mounted not into `/dev/..` but `<host_fs>/dev/...`
Possible fix would be either to remove the block (preferable as symlink contains the name of LVM and it is not making sense to eval it to the name of device) or to change the above-mentioned line  line to:
`devpath, err := filepath.EvalSymlinks(common.GetEnv("HOST_MOUNT_PREFIX","/",d.Device))`

**Comment by i-prudnikov** [comment]
Opened a pull request to fix that: https://github.com/shirou/gopsutil/pull/819

**Comment by i-prudnikov** [comment]
@danielnelson, the proposed fix is merged: https://github.com/shirou/gopsutil/pull/819

**Comment by danielnelson** [comment]
The upstream fix should be included in 1.14.0.
